Noti/PushManager.swift

```diff
@@ -305,7 +314,9 @@ class PushManager: NSObject, WebSocketDelegate, NSUserNotificationCenterDelegate
         
         if let type = message["type"].string {
             switch type {
-                
+            case "nop":
+                self.nopTimer.invalidate()  // Resetting nopTimer
+                self.nopTimer = Timer.scheduledTimer(timeInterval: 35.0, target: self, selector: #selector(PushManager.nopTimeout), userInfo: nil, repeats: false)
```

Comment:
I think it'd be better if this selector targets `disconnect`. Cuts down on duplicate code.
We could give it a `attemptReconnect` parameter which is `false` by default like so:
```swift
@objc internal func disconnect(attemptReconnect: Bool = true) {
if !attemptReconnect {
    self.killed = true
}
```
We'd have to make sure all existing calls to disconnect call it as so: `self.disconnect(attemptReconnect: false)`